meta-lmp-base/classes/lmp.bbclass

```diff
@@ -19,8 +19,13 @@ sota_var_cleanup() {
 	if ${@bb.utils.contains('IMAGE_FSTYPES', 'ota-ext4', 'true', 'false', d)}; then
 		# Remove /var stuff (ignored by ostree)
 		cd ${IMAGE_ROOTFS}/var
-		rmdir -v backups spool local lib/misc
-		rmdir -v volatile ${@'' if oe.types.boolean('${VOLATILE_LOG_DIR}') else 'log'}
+		ostree_rmdir_helper backups
+		ostree_rmdir_helper spool
+		ostree_rmdir_helper local
+		ostree_rmdir_helper lib/misc
+		ostree_rmdir_helper volatile
+		if not oe.types.boolean('${VOLATILE_LOG_DIR}'):
```

Comment:
This won't parse correctly as it is shell and not a python code block.

Reply:
right, it fails to parse. fixed and tested locally and `ostree_rmdir_helper` works with arguments